src/GCRealTimeMon/Utilities/PrintUtilities.cs

```diff
@@ -48,7 +71,7 @@ public static string GetLineSeparator(Configuration.Configuration configuration)
             }
 
             string lineSeparator = "";
-            for(int i = 0; i < repeatCount; i++)
+            for (int i = 0; i < repeatCount; i++)
```

Comment:
> for (int i = 0; i < repeatCount; i++) [](http://example.com/codeflow?start=11&length=38)

nit - I guess your formatting is to have a space for the for/foreach statement, I do notice some inconsistency in the src - we should standize the formating. is there any conventional formatting, is this "space before '(' unless it's a method call" a convention in VS? if there's already something we use that and reformat our src.

Reply:
VS has this setting which would apply to all keywords not just for/foreach. Based on the source history I'm thinking you also have it set to enabled Maoni? I'm pretty sure it's enabled by default in VS:
![image](https://user-images.githubusercontent.com/578591/149262478-e311d99a-8fbd-46a7-b762-a184bcb4a10b.png)

@MokoSan this commit was the first I found of the "no space between keywords" - is that your preference or perhaps it was a misconfiguration? https://github.com/Maoni0/realmon/commit/84080aaf4a051031b496bab09bbe97988692bc40